### DISCUSSION TITLE
Finish the first line

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#766] [Galaxy Trivia] [answered]
*"In a hole in the ground there lived a ..."*

**Bonus:** List the source and author.

**Comment by peterbaccaro** [comment, marked as answer]
"In a hole in the ground there lived a **hobbit**." These are the opening words of **J.R.R. Tolkien**'s novel "**The Hobbit**". The book tells the story of Bilbo Baggins, a hobbit who is enlisted by a wizard named Gandalf to help a group of dwarves reclaim their homeland from a dragon named Smaug.
